CruzHacks/cruzhacks-2021-website
# Milestones component.

Problem
=======
As a hacker, 
I want to see CH milestones,
So I can see how CH has done in the past.

[Link to Asana Ticket](https://app.asana.com/0/1197738625416321/1199333062646019)

Solution
========
- In order to create the cyan-red effect for the title texts, I created the `anaglyph-title-text` mixin which can be used by the Missions, Milestones, and Topics components. 
- I also converted the background grid to png so it can be repeated vertically. 
- Each Milestone lives inside its own `Milestones__box` div in a container. Flex wrap is allowed so the milestones can wrap onto multiple lines on smaller screens.


Change summary:
---------------
- Add Milestones component.

Dev-Ops
=======
If you added an ENV variable, please check off that you've updated the following  
- [ ] `.env` & `.env.sample` 
- [ ] `webpack.config.js` 
- [ ] GCP Secret Manager
- [ ] Github Secrets 
- [ ] GithubActions Workflow

Steps to Verify:
----------------
1. Checkout branch and start local server.
2. Verify Milestones component is present & responsive in home page.

Screenshots (optional):
-----------------------
<img src="https://user-images.githubusercontent.com/20177171/101215192-3d524a80-3632-11eb-9af0-441a12532afa.png" width="500" />
